ICAFIX/ReApplyFixPipeline.sh

```diff
@@ -663,7 +686,11 @@ main()
 			$FSLDIR/bin/imrm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}
 		fi
 		$FSLDIR/bin/imrm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/filtered_func_data
-        rm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/Atlas_hp_preclean.dtseries.nii
+		#TSC: should these be ${hpStr} or _hp${hp}?
+		rm -f "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/Atlas_hp_preclean.dtseries.nii" "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.nii.gz"
+	else
+		#don't leave this file with a hard to interpret name - again, hpStr?
+		mv -f "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/Atlas_hp_preclean.dtseries.nii" "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_Atlas_hp${hp}.dtseries.nii"
```

Comment:
For the same reasons as above, this needs to be conditional on hp>=0 (otherwise we are creating a file named "_hp${hp}" that wasn't actually filtered.

Reply:
See 4eedd5ed883a2e2972bae49e4abaee5394c43c2d.